daprdocs/content/en/js-sdk-docs/js-actors/_index.md

```diff
@@ -95,10 +95,16 @@ const clientOptions = {
   },
 };
 
+const serverOptions = {
+  serverHost: serverHost,
+  serverPort: serverPort,
+  clientOptions: clientOptions,
+};
+
 // Use the options when creating DaprServer and DaprClient.
 
 // Note, DaprServer creates a DaprClient internally, which needs to be configured with clientOptions.
-const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, clientOptions);
+const server = new DaprServer(serverOptions);
```

Comment:
```suggestion
const server = new DaprServer({serverHost, serverPort, clientOptions});
```
Same change everywhere else, as suggested to me by @XavierGeerinck

Reply:
Done